### PR TITLE
[PLT-7578] Fix broken UI for expired email invite link

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix broken UI for expired email invite link

#### Ticket Link
Jira ticket: [PLT-7578](https://mattermost.atlassian.net/browse/PLT-7578)

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

**Comment by hmhealey** [comment]
Moving this back to PM review since it involves UI changes that weren't specced

**Comment by jasonblais** [comment]
I tested and the error page matches the expected case in the ticket (https://mattermost.atlassian.net/browse/PLT-7578)

image

@hmhealey Please ping me if something else was changed in the UI that PMs should be aware of? I didn't find anything in particular.

**Comment by hmhealey** [comment]
@jasonblais I wasn't expecting this page to show "Create an account with..." or "Already have an account?" when it has an error since it's not a signup page, and it feels out of place (2/5). I'm fine if we want it to look like a signup page though.

File: components/do_verify_email.jsx [hunk 1]
```diff
@@ -20,45 +21,91 @@ export default class DoVerifyEmail extends React.Component {
...
+                            <div className='margin--extra'>
+                                <h5>
+                                    <strong>
+                                        <FormattedMessage
+                                            id='signup.title'
+                                            defaultMessage='Create an account with:'
+                                        />
+                                    </strong>
+                                </h5>
+                            </div>
```
Comment (ccbrown) [inline components/do_verify_email.jsx]: Shouldn't you put the signup controls here?

**Comment by ccbrown** [comment]
@jasonblais I agree with @hmhealey and really think we should either remove the "Create an account with:" header or actually add the signup controls to this page. It just looks broken as-is.

**Comment by jasonblais** [comment]
That makes sense. @saturninoabril can you help with that?

**Comment by saturninoabril** [comment]
@jasonblais Noted, see updated patch.

**Pull request merged**